### PR TITLE
Chain1Tests refactored using utils. Util sendSubstring() refactored.

### PR DESCRIPTION
**Pull request by ArinaJur**
Chain1Tests refactored using utils. 
Util sendSubstring() refactored.

https://trello.com/c/UCfnmOfp/948-entity-eventschain1test-refactoring-with-random-values

### CODE REVIEW

**Review by guram21 — APPROVED** [review]

File: src/test/java/utils/TestUtils.java [hunk 1]
```diff
@@ -122,8 +119,7 @@ public static String currentDateTime(){
         return new SimpleDateFormat("dd/MM/yyyy HH:mm:ss").format(new Date());
     }
 
-    public static String decimalFormat(double decimal)
-    {
+    public static String decimalFormat(double decimal) {
```
Comment (oooleks) [inline src/test/java/utils/TestUtils.java:122]: Method name should start with a verb.

File: src/test/java/test/entity/EntityEventsChain1Test.java [hunk 2]
```diff
@@ -150,15 +149,15 @@ public void createEmptyRecord() {
...
-                .clickNewFolder();
-        ErrorPage errorPage = chain1EditPage
-                .setValue(chain1EditPage.getF1(), LARGE_INT)
-                .clickSaveButtonErrorExpected();
+                .clickNewFolder()
+                .setValue(LARGE_INT);
+
+        ErrorPage errorPage = chain1EditPage.clickSaveButtonErrorExpected();
 
         Assert.assertEquals(errorPage.getErrorMessage(), ProjectConstants.ERROR_MESSAGE);
```
Comment (oooleks) [inline src/test/java/test/entity/EntityEventsChain1Test.java:162]: Could be simplified as: 
`       Assert.assertEquals(chain1EditPage.clickSaveButtonErrorExpected().errorPage.getErrorMessage(), ProjectConstants.ERROR_MESSAGE);`

File: src/test/java/test/entity/EntityEventsChain1Test.java [hunk 3]
```diff
@@ -103,21 +101,22 @@ public void deleteRecord() {
...
-
-        chain1EditPage.setValue(chain1EditPage.getF1(), DECIMAL);
+                .clickNewFolder()
+                .setDecimalValue(DECIMAL);
 
-        //Temporary added for tracking result on Travis
-        System.out.println(DECIMAL);
+        /* Temporary added for tracking result on Travis */
+        System.out.println(chain1EditPage.getExpectedDecimals(DECIMAL));
         System.out.println(chain1EditPage.getInputValues());
```
Comment (oooleks) [inline src/test/java/test/entity/EntityEventsChain1Test.java:113]: Do not forget to delete before merge into main

**Review by GalinaRuban — APPROVED** [review]

File: src/test/java/model/entity/edit/Chain1EditPage.java [hunk 4]
```diff
@@ -105,25 +95,8 @@ public boolean isFocusOnF10() {
         double f = Double.parseDouble(f1);
         for (int i = 1; i < 10; i++) {
             f *= 2;
-            expectedValues.add(String.valueOf(fmt(f)));
+            expectedValues.add(String.valueOf(decimalFormat(f)));
```
Comment (oooleks) [inline src/test/java/model/entity/edit/Chain1EditPage.java:98]: Should you rename it as well?

**Pull request merged**